Docs/source/dataanalysis/workflows/tunneling.rst

```diff
@@ -0,0 +1,55 @@
+.. _dataanalysis-workflows-tunneling:
+
+Port Tunneling
+==============
+
+SSH port tunneling (port forwarding) is a secure way to access a computational service of a remote computer.
+A typical workflow where you might need port tunneling is for Jupyter data analysis, e.g., when analyzing data on your desktop computer but working from your laptop.
+
+Before getting started here, please note that many HPC centers offer a pre-installed Jupyter service, where tunnel is **not** needed.
+For example, see the :ref:`NERSC Jupyter <post-processing-perlmutter>` and :ref:`OLCF Jupyter <post-processing-frontier>` services.
+
+
+.. _dataanalysis-workflows-tunneling-background:
+
+Introduction
+------------
+
+When running a service such as Jupyter from your command line, it will start a local (web) port.
+The IPv4 address of your local computer is always ``127.0.0.1`` or the alias ``localhost``.
+
+As a secure default, you cannot connect from outside your local computer to this port.
+This prevents misconfigurations where one could, in the worst case, connect to your open port without authentication and execute commands with your user privileges.
+
+One way to access your remote Jupyter desktop service from your laptop is to forward the port started remotely via an encrypted SSH connection to a local port on your current laptop.
+The following section will explain the detailed workflow.
+
+
+.. _dataanalysis-workflows-tunneling-workflow:
+
+Workflow
+--------
+
+* you connect via SSH to your desktop at work, in a terminal (A) as usual
+
+  * e.g., ssh ``username@your-computers-hostname.dhcp.lbl.gov``
+  * you start Jupyter locally in headless mode, it will show you a ``127.0.0.1`` (aka ``localhost``) URL, by default on port TCP ``8888``
```

Comment:
```suggestion
  * you start Jupyter locally in headless mode, e.g., `jupyter lab --no-browser`
  * this will show you a ``127.0.0.1`` (aka ``localhost``) URL, by default on port TCP ``8888``
```